### ISSUE TITLE
Not working in macOS Ventura

### ISSUE THREAD

**Issue by rogeruiz** [keycastr/keycastr#252]
hi @akitchen I've tried these steps you've outlined with macOS Ventura. I can't seem to get Keycastr working though.

[=> Steps you've outlined in #222](https://github.com/keycastr/keycastr/issues/222#issuecomment-1008110225) (https://github.com/keycastr/keycastr/issues/222#issuecomment-1008110225)

I'd also like to submit a Ventura specific screenshot and documentation PR eventually to help out too since it looks like it'll be needed since Apple changed _System Preferences_ to _System Settings_ in Ventura. 

Also, I'll submit a PR with the changes you mentioned in your message separately for folks who aren't on Ventura yet.

### system info

MacBook Pro 16-inch, 2019
macOS 13.0 (22A380)

**Comment by rogeruiz** [comment]
I found the Wiki, and then tried looking for the _Enable access for assistive devices_ in the Apple Support documentation for Ventura and haven't found anything yet. It looks like it might be covered under the Accessibility section of the new System Settings under Privacy and Security. This leads me to believe maybe there's an issue with the code. I have the project checked out and am going to take a look at the source to see if there is some kind of error or bug that I am not seeing from the running application.

[=> Welcome to the keycastr wiki!](https://github.com/keycastr/keycastr/wiki)

[=> Change Keyboard settings for accessibility on Mac](https://support.apple.com/guide/mac-help/change-keyboard-settings-for-accessibility-mchlae61a6de/13.0/mac/13.0)

[=> Change Accessibility settings on Mac](https://support.apple.com/guide/mac-help/change-accessibility-settings-mchlp1400/mac)

**Comment by akitchen** [comment]
Hello and thank you for opening an issue and accompanying PR!

The wiki is something I haven't paid attention to in a long time, thanks for mentioning that. Will need to give it a bit more thought since it's Yet Another Thing that would need to be kept up to date.

Sorry to hear about the issues with Ventura. Unfortunately I'm not able to run it on my main machine yet but I do have it in a VM. I will try to take a look soon, and of course any further PRs are welcome if you discover anything while looking into the code yourself.

**Comment by akitchen** [comment]
So, I was able to run the current build of KeyCastr in a Ventura VM. It may have had a delay in initializing itself in the user defaults since it didn't seem to work at first, but I switched on the mouse visualizer and it worked, and when I switched over to Svelte it worked too. And then when switching back to the Default visualizer it worked OK from there on out. 

I'm still on a beta release of Ventura and for some reason resetting the user defaults doesn't seem to work so I can't reset my state and do it again. However, I do think this issue is recoverable if you switch to the Svelte visualizer and then back to Default. Give it a try and let us know!

It's also possible that re-compiling keycastr as-is (when I get to the next release) will pick up other implicit fixes and improvements in the macOS SDKs but again, I was able to get the current build to run OK.
